### PR TITLE
Initial docs for Encryption Password

### PR DESCRIPTION
**Pull request by petems**
WIP docs for the Encryption Password

### CODE REVIEW

File: content/source/docs/enterprise/private/encryption-password.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# Private Terraform Enterprise Encryption Password
+
+During the installation of Private Terraform Enterprise, a password is used to encrypt sensitive information at rest. The default value is auto-generated, but we strongly suggest you create your own password.
+
+Be sure to retain the value, because you will need to use this
+password to restore access to the data in the event of a reinstall.
+
+The Encryption Password is used to protect the vault unseal
+key and root token when the internal Vault is used.
+It allows us to store those details in Postgresql,
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/encryption-password.md]:
```suggestion
It allows us to store those details in PostgreSQL,
```

File: content/source/docs/enterprise/private/encryption-password.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+which makes vault “stateless” in that it only depends on details in Postgres and the encryption password.
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/encryption-password.md]:
```suggestion
which means that Vault is only dependent on the encryption password itself and details in PostgreSQL.
```

File: content/source/docs/enterprise/private/encryption-password.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## Specifying the Encryption Password
+
+The password can be specified during an unattended installation with the installer option `enc_password` within the [application settings](#application-settings) file:
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/encryption-password.md]: This link is not correct, as the application settings are mentioned on a different page - can you make sure the link goes to that page?

File: content/source/docs/enterprise/private/install-installer.html.md [hunk 4]
```diff
@@ -198,7 +198,7 @@ From a shell on your instance, in the directory where you placed the `replicated
    and re-run the checks, or ignore the warnings and proceed. If the system is running behind a proxy and is unable to connect to `releases.hashicorp.com:443`, it is likely safe to proceed; this check does not currently use the proxy. For any other issues, if you proceed despite the warnings, you are assuming the support responsibility.
 1. Set an encryption password used to encrypt the sensitive information at rest. The default value is auto-generated,
    but we strongly suggest you create your own password. Be sure to retain the value, because you will need to use this
-   password to restore access to the data in the event of a reinstall.
+   password to restore access to the data in the event of a reinstall. See [Encryption Password](./encryption-password.html#encryption-password) for more information on the Encryption Password.
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/install-installer.html.md]:
```suggestion
   password to restore access to the data in the event of a reinstall. See [Encryption Password](./encryption-password.html#encryption-password) for more information on the encryption password.
```

File: content/source/docs/enterprise/private/encryption-password.md (hunk 3 above)
Comment (petems) [reply content/source/docs/enterprise/private/encryption-password.md]: Fixed 👍

**Comment by petems** [comment]
@armchairlinguist Fixed the issues, tried to make the page breaks consistant, fixed links. Can I get a 👍? 😄

File: content/source/docs/enterprise/private/encryption-password.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
+---
+layout: "enterprise2"
+page_title: "Private Terraform Enterprise Encryption Password"
+sidebar_current: "docs-enterprise2-private-encryption-password"
+---
+
+# Private Terraform Enterprise Encryption Password
+
+During the installation of Private Terraform Enterprise,
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/encryption-password.md]: This is awfully short! I think the standard is 80. But it is consistent so I'm fine with it. :)

File: content/source/docs/enterprise/private/install-installer.html.md [hunk 6]
```diff
@@ -198,7 +198,7 @@ From a shell on your instance, in the directory where you placed the `replicated
    and re-run the checks, or ignore the warnings and proceed. If the system is running behind a proxy and is unable to connect to `releases.hashicorp.com:443`, it is likely safe to proceed; this check does not currently use the proxy. For any other issues, if you proceed despite the warnings, you are assuming the support responsibility.
 1. Set an encryption password used to encrypt the sensitive information at rest. The default value is auto-generated,
    but we strongly suggest you create your own password. Be sure to retain the value, because you will need to use this
-   password to restore access to the data in the event of a reinstall.
+   password to restore access to the data in the event of a reinstall. See [encryption password](https://www.terraform.io/docs/enterprise/private/encryption-password.html) for more information.
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/install-installer.html.md]:
```suggestion
   password to restore access to the data in the event of a reinstall. See [Encryption Password](https://www.terraform.io/docs/enterprise/private/encryption-password.html) for more information.
```

Sorry, I didn't mean to suggest the link should be downcased - it's a page title, so it makes sense to title case it. It's just the entity itself that we prefer lowercase for.

File: content/source/docs/enterprise/private/encryption-password.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+encryption password itself and details in PostgreSQL.
+
+## Specifying the Encryption Password
+
+The password can be specified during an unattended
+installation with the installer option `enc_password`
+within the [application settings JSON file when
+using the automated install procedure](https://www.terraform.io/docs/enterprise/private/automating-the-installer.html#available-settings):
+
+```
```
[the remark this answers is not in the document]
Comment (petems) [reply content/source/docs/enterprise/private/encryption-password.md]: Fixed 👍

**Comment by petems** [comment]
🚢 🇮🇹

**Pull request merged**